src/components/ChatEntry.js

```diff
@@ -1,22 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const { sender, body, timeStamp } = props;
+  const [liked, setLiked] = useState(false);
```

Comment:
It's React best practice to group the state variables at the top of the component so it's easy to find like you do here for `liked` and `setLiked`. Good job!